.github/workflows/test.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     needs: lint
     strategy:
       matrix:
-        ckan-version: ['2.10', '2.10.1']
+        ckan-version: ['2.10']
```

Comment:
should we leave this as `2.10.1` so that test gets run?

Reply:
2.10 is not `2.10.0`. 2.10 means 2.10.*. Any reason we want to stop and pin at old 2.10.1?

Reply:
so if OKFN releases 2.10.2, this will pull that tag?

it does look like 2.10 and 2.10.1 are using the same hash.

https://hub.docker.com/r/openknowledge/ckan-dev/tags

Reply:
if that's the case then i just need to make `CKAN 2.10.1` action not required.

Reply:
yes, remove 2.10.1 from the code and requirement.